plutus-benchmark/validation/Main.hs

```diff
@@ -70,7 +70,7 @@ getAppliedScript progName validatorNumber datumNumber redeemerNumber contextNumb
   redeemer  <- loadScript "Redeemer"  redeemerNumber
   context   <- loadScript "Context"   contextNumber
   let appliedValidator = validator `UPLC.applyProgram` datum `UPLC.applyProgram` redeemer `UPLC.applyProgram` context
-  pure $ void . UPLC.toTerm $ appliedValidator
+  pure $ void . UPLC.toTerm $ UPLC.globalifyProgram $ appliedValidator
```

Comment:
If I've got the Criterion stuff right then we're not even including the cost of globalification in the benchmarking times, and as we've seen elsewhere traversal of the entire AST seems to be expensive.  That's not very encouraging.